src/applications/travel-pay/components/TravelClaimCard.jsx

```diff
@@ -14,44 +15,55 @@ const CLAIM_STATUS = {
   CLOSED: 'Closed',
 };
 
+function parseDatetime(datetimeString) {
+  const dateTime = new Date(datetimeString);
+  const formattedDate = format(dateTime, 'eeee, MMMM d, yyyy');
+  const formattedTime = format(dateTime, 'h:mm a');
+
+  return [formattedDate, formattedTime];
+}
+
 export default function TravelClaimCard(props) {
   const {
     id,
     createdOn,
     claimStatus,
-    claimName,
+    // claimName,
     claimNumber,
-    appointmentName,
-    appointmentDate,
+    // appointmentName,
+    appointmentDate: appointmentDateTime,
+    appointmentLocation,
```

Comment:
We also need the clinic name, but I forget - is that represented in `appointmentLocation` here?